core/src/test/java/org/wildfly/extras/sunstone/api/impl/docker/DockerNodeTest.java

```diff
@@ -535,6 +538,23 @@ public void testInsufficientCapabilities() {
         }
     }
 
+    @Test
+    public void testVolumeBindings() {
+        final int nonce = new Random().nextInt();
+        try (DockerNode node = (DockerNode) dockerProvider.createNode("alpineVolumes",
+                ImmutableMap.<String, String> of("template", "alpine", "docker.binds", "/tmp:/data"))) {
+            final ExecResult execResult = node.exec("sh", "-c", "echo " + nonce + " > /data/testVolumeBindings");
```

Comment:
Nope. It would be nice, but this node doesn't have SSH server. Hopefully we'll be able to workaround it in the future.